l10n_it_corrispettivi_fatturapa_out/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Questo modulo integra le funzionalità dei corrispettivi con la generazione della fattura elettronica.
+
+**English**
+
+This module integrates corrispettivi and electronic invoicing.
```

Comment:
corrispettivi → sums due
come da glossario
https://www.agenziaentrate.gov.it/wps/portal/english/glossary

Forse meglio anche uniformare il testo inglese con quello italiano o viceversa.

Reply:
@primes2h 'sums due' mi sembra un pò troppo generico, comunque meglio che lasciare 'corrispettivo' ovunque.
Forse i corrispettivi sono le ricevute? @eLBati cosa ne dici?
Se così fosse potrei usare 'receipt'

Reply:
> @primes2h 'sums due' mi sembra un pò troppo generico, comunque meglio che lasciare 'corrispettivo' ovunque.

Un'alternativa potrebbe essere "amount(s) due".
Però prima è necessario definire bene il contesto.

Reply:
Ho trovato questo documento, sempre dell'Agenzia delle Entrate, nel quale traducono "Fatture e corrispettivi" con "Invoices and payments".


https://www.agenziaentrate.gov.it/wps/file/Nsilib/Nsi/Schede/Comunicazioni/Fatture+e+corrispettivi/Fatture+e+corrispettivi+ST/ST+invio+di+fatturazione+elettronica/ST+Fatturazione+elettronica+-+Allegato+A+inglese/Appendix+A_Technical+Specifications+vers+1.3.pdf

Reply:
Il modulo `l10n_it_corrispettivi` nasce effettivamente per le ricevute (per gli scontrini c'è il POS ed i suoi moduli), quindi quel termine mi sembra il più appropriato (eviterei però la modifica nel nome dei moduli e nei campi DB, altrimenti dovremmo anche fare gli script di migrazione DB)

Reply:
Devo dire che in effetti non si capiva cosa cosa fossero questi corrispettivi. (il modulo non l'avevo provato però).

Usare il termine corretto "ricevute/receipts" modifica (in meglio) la situazione.
Dovrebbe essere modificato sia nelle stringhe di traduzione sorgente che nel readme.

Aggiorno di conseguenza la review delle altre PR dei moduli collegati.

Reply:
Modificato Corrispettivi in ricevute e simili.